src/visualizers/Chaos.ts

```diff
@@ -366,12 +363,14 @@ class Chaos extends P5Visualizer(paramDesc) {
 
     draw() {
         const sketch = this.sketch
-        // we do pixelsPerFrame pixels each time through the draw cycle;
-        // this speeds things up essentially
+        // We attempt to do pixelsPerFrame pixels each time through the
+        // draw cycle; this speeds things up essentially. Note that we
```

Comment:
I feel like I'm probably originally to blame for the word "essentially" here, but I'm not sure what it means so maybe we should leave it out?

Reply:
OK, since I touched that comment anyway I will re-word it.